bundles/org.openhab.persistence.influxdb2/src/main/java/org/openhab/persistence/influxdb2/internal/InfluxDB2PersistenceService.java

```diff
@@ -208,27 +225,31 @@ public void store(Item item, @Nullable String alias) {
     @Reference(cardinality = ReferenceCardinality.OPTIONAL, policy = ReferencePolicy.DYNAMIC)
     protected void setItemRegistry(ItemRegistry itemRegistry) {
         this.itemRegistry = itemRegistry;
-        if (logger.isTraceEnabled())
+        if (logger.isTraceEnabled()) {
             logger.trace("ItemRegistry has been set");
+        }
     }
 
     protected void unsetItemRegistry(ItemRegistry itemRegistry) {
         this.itemRegistry = null;
-        if (logger.isTraceEnabled())
+        if (logger.isTraceEnabled()) {
             logger.trace("ItemRegistry has been unset");
+        }
     }
 
     @Reference(cardinality = ReferenceCardinality.OPTIONAL, policy = ReferencePolicy.DYNAMIC)
     protected void setMetadataRegistry(MetadataRegistry metadataRegistry) {
         this.metadataRegistry = metadataRegistry;
-        if (logger.isTraceEnabled())
+        if (logger.isTraceEnabled()) {
             logger.trace("MetadataRegistry has been set");
+        }
     }
 
     protected void unsetMetadataRegistry(MetadataRegistry metadataRegistry) {
         this.metadataRegistry = null;
-        if (logger.isTraceEnabled())
+        if (logger.isTraceEnabled()) {
```

Comment:
You can remove all these if statements whenever the logging doesn't do any expensive operations (like string concatenations). That makes the code a more readable and compact.